js/script.js

```diff
@@ -8,6 +8,51 @@
       });
   })
 
+  var npmService = [
+    'https://api.npmjs.org/downloads/point/last-day/gulp'
+  ].map(function (url) {
+    return fetch(url)
+      .then(function(resp) {
+        return resp.json();
+      });
+  })
+
+  var pluginService = [
+    'https://api.npms.io/v2/search?q=gulp'
```

Comment:
we should use our plugin search for this because we actually want to check the `gulpplugin` keyword, not gulp

Reply:
Ok I can get the keyword updated with`gulpplugin` this evening.

Reply:
Corrected